File: core/src/main/java/com/github/dimaslanjaka/helpers/core/Service/Activity.java
package com.dimaslanjaka.tools.Helpers.core.Service;

import android.content.Context;
import android.content.Intent;

public class Activity {
  /**
   * start/switch activity
   *
   * @param packageContext from activity
   * @param cls            to activity
   */
  public static void start(Context packageContext, Class<?> cls) {
    Intent myIntent = new Intent(packageContext, cls);
    myIntent.putExtra("user", "value"); //Optional parameters
    //packageContext.this.startActivity(myIntent);
    packageContext.startActivity(myIntent);
  }
}

File: library-android/src/main/java/android/Activity.kt
package android

import android.content.Intent

@Suppress("unused")
object Activity {
    @JvmStatic
    fun start(clazz: Class<*>, extras: Map<String, String>) {
        val intent = Intent(App.globalContext(), clazz)
        intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK)
        for ((key, value) in extras) {
            println("EXTRAS $key = $value")
            intent.putExtra(key, value)
        }
        App.globalContext().startActivity(intent)
    }

    @JvmStatic
    fun start(clazz: Class<*>) {
        start(clazz, mapOf())
    }
}